examples/nrf_802154_delayed_echo_client/src/nrf_802154_delayed_echo_client.c

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 2018 - 2020, Nordic Semiconductor ASA
+ * All rights reserved.
+ *
```

Comment:
Missing SPDX tag. Applies to source files of the remaining examples as well

Reply:
Done.